backend/gsr_booking/management/commands/send_gsr_reminders.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from gsr_booking.models import Reservation
+from user.models import SERVICE_GSR_BOOKING, NotificationToken
+from user.notifications import send_push_notif
+
+
+class Command(BaseCommand):
+    help = "Sends reminders for the GSR Bookings."
+
+    def handle(self, *args, **kwargs):
+        # iterate through all reservations scheduled for the next 30 minutes
+        for reservation in Reservation.objects.filter(
+            start__gt=timezone.now(), start__lte=timezone.now() + datetime.timedelta(minutes=30)
```

Comment:
for this query, if you could change to within 10 minutes instead of 30, as we will be notifying uses on the 20th and 50th minute every hour. also, you can also filter for `is_cancelled=False` instead of doing the if statement on line 19